src/components/supporters/index.js

```diff
@@ -2,7 +2,7 @@ import React from "react";
 import SectionContainer from "../sectionContainer";
 import Translate from "@docusaurus/Translate";
 import "./index.scss";
-
+import Link from "@docusaurus/Link";
 const supportList = [
```

Comment:
@coder12git  Thank you for your contributions!  I think adding a blank line here between import and following parts would make the format look better.

Reply:
> @coder12git Thank you for your contributions! I think adding a blank line here between import and following parts would make the format look better.

Done @Shelley-BaoYue